test/example/deployment.yaml

```diff
@@ -1,6 +1,18 @@
 # SPDX-FileCopyrightText: The RamenDR authors
 # SPDX-License-Identifier: Apache-2.0
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginxservice
+  labels:
+    app: nginx
+spec:
+  ports:
+  - port: 80
```

Comment:
This conflicts with the ingress port - we can use another port. We also need
to specify targetPort with the name of the example app port:

    - port: 8000
      name: example-port